int64 id
float64[] pos_x
float64[] pos_y
float64 radius